include/boost/spirit/home/x3/directive/skip.hpp

```diff
@@ -67,16 +71,48 @@ namespace boost { namespace spirit { namespace x3
           , skipper(skipper)
         {}
 
-        template <typename Iterator, typename Context
-          , typename RContext, typename Attribute>
+        template <typename Iterator, typename RContext, typename Attribute>
```

Comment:
You just removing `skip[]` directive for noexcept mode? I don't understand why.

Reply:
My understanding is that `skip[]` is implemented in `reskip_directive::parse()`, not in this specialization.
Instead this one is `skip_directive::parse()`, and this specialization is very specifically called when the parent context was `unused_type`. We need to treat it as a logic error when it's non-throwing mode, thus `static_assert(false)`.

Reply:
This issue has been resolved in https://github.com/boostorg/spirit/compare/d63d3a4fbce11e1fb8d8c91a8b874e983000d437..88d9987167799e009427b8ec48154809d8db37dc